packages/middleware-io/src/io.event.middleware.ts

```diff
@@ -36,6 +36,8 @@ export const eventValidator$ = <U extends Schema>(schema: U, options?: Validator
       )),
     ) as Observable<ValidationResult<U>>;
 
-  return (event$: Observable<Event>) =>
-    event$.pipe(mergeMap(validate));
+  return (input: Observable<Event> | Event) =>
+    isObservable(input)
```

Comment:
This is definitely "convinient", but its a bit weird to see a function, conventionally postfixed with `$`, which takes something other than an Bbservable. :thinking:

I don't disagree with it, though.

Reply:
IMO it is not a problem, the function always returns `Observable` 🤔